libs/components/src/lib/indoor-maps.component.ts

```diff
@@ -0,0 +1,401 @@
+import {
+    Component,
+    ViewChild,
+    ElementRef,
+    OnInit,
+    Input,
+    SimpleChanges,
+} from '@angular/core';
+import { AsyncHandler, HashMap } from '@placeos/common';
+import { ViewerStyles, ViewAction } from '@placeos/svg-viewer';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+declare let mapsindoors: any;
+
+interface GeolocationCoordinates {
+    latitude: number;
+    longitude: number;
+    altitude: number | null;
+    accuracy: number;
+    altitudeAccuracy: number | null;
+    heading: number | null;
+    speed: number | null;
+}
+
+interface GeolocationPosition {
+    coords: GeolocationCoordinates;
+    timestamp: number;
+}
+
+@Component({
+    selector: 'indoor-maps',
+    template: `
+        <div id="map" class="absolute inset-0 flex items-center justify-center">
```

Comment:
Make this `id` attribute value more unique than just "map" to prevent any clashes.
`id` attribute should be unique per document. You can't guarantee the uniqueness of a simple value.

Reply:
Changed to a more unique id value.